cmd/broker/main.go

```diff
@@ -482,17 +482,27 @@ func createAPI(router *mux.Router, servicesConfig broker.ServicesConfig, planVal
 	fatalOnError(err, logs)
 	logs.Infof("Number of globalAccountIds for unlimited freeemium: %d\n", len(freemiumGlobalAccountIds))
 
+	var convergedCloudRegionProvider = broker.NewOneForAllConvergedCloudRegionsProvider()
+
+	if cfg.SapConvergedCloudRegionMappingsFilePath != "" {
+		convergedCloudRegionProvider, err = broker.NewDefaultConvergedCloudRegionsProvider(cfg.SapConvergedCloudRegionMappingsFilePath, &broker.YamlRegionReader{})
+		fatalOnError(err, logs)
+		logs.Info("sap-converged-cloud plan region mappings loaded")
+	} else {
+		logs.Warn("sap-converged-cloud plan region mappings file path is not set, using old configuration")
```

Comment:
this defaulting is dangerous that could cause that misconfigured production could the in BTP Australia return eu-de1

Reply:
This was introduced for backward compatibility for tests. It was easier not to modify old tests in this case. We agreed to change that later on.